actions/access_keys/revoke.go

```diff
@@ -15,8 +15,10 @@ import (
 // @Description	Revoke access key
 // @Tags		Access-key
 // @Produce		json
-// @Param		id query string true "id"
-// @Success		201
+// @Param		id query string true "id of the access key"
+// @Success		200	{object} models.AccessKey "Created AccessKey"
```

Comment:
model.AccessKey can be removed from revoke response

Reply:
I change the description to revoked AccessKey but still we should return the model because If I change it here we should change it also in all clients. Maybe I should add new task to check and remove models from all revoke endpoints